EvidenceApi/V1/UseCase/CreateDocumentSubmissionWithoutEvidenceRequestUseCase.cs

```diff
@@ -0,0 +1,126 @@
+            _documentsApiGateway = documentsApiGateway;
+            _staffSelectedDocumentTypeGateway = staffSelectedDocumentTypeGateway;
+        }
+
+        public async Task<DocumentSubmissionWithoutEvidenceRequestResponse> ExecuteAsync(DocumentSubmissionWithoutEvidenceRequestRequest request)
+        {
+            ValidateRequest(request);
+
+            Claim claim;
+            S3UploadPolicy createdS3UploadPolicy;
+
+            try
+            {
+                var claimRequest = BuildClaimRequest(request);
+                claim = await _documentsApiGateway.CreateClaim(claimRequest);
+                createdS3UploadPolicy = await _documentsApiGateway.CreateUploadPolicy(claim.Document.Id);
+            }
+            catch (DocumentsApiException ex)
+            {
+                throw new BadRequestException($"Issue with DocumentsApi so cannot create DocumentSubmission: {ex.Message}");
+            }
+
+            var documentSubmission = BuildDocumentSubmission(request, claim);
+            var createdDocumentSubmission = new DocumentSubmission();
+            try
+            {
+                createdDocumentSubmission = _evidenceGateway.CreateDocumentSubmission(documentSubmission);
+            }
+            catch (Exception)
+            {
+                throw new BadRequestException($"A resident with ID {request.ResidentId} does not exist.");
+            }
+
+            var staffSelectedDocumentType = _staffSelectedDocumentTypeGateway.GetDocumentTypeByTeamNameAndDocumentTypeId(request.Team, documentSubmission.StaffSelectedDocumentTypeId);
+
+            return createdDocumentSubmission.ToResponse(staffSelectedDocumentType, createdS3UploadPolicy, claim);
+        }
+
+        private static ClaimRequest BuildClaimRequest(DocumentSubmissionWithoutEvidenceRequestRequest request)
+        {
+            var claimRequest = new ClaimRequest()
+            {
+                ServiceAreaCreatedBy = request.Team,
+                UserCreatedBy = request.UserCreatedBy,
+                ApiCreatedBy = "evidence_api",
+                RetentionExpiresAt = DateTime.UtcNow.AddMonths(3).Date,
+                ValidUntil = DateTime.UtcNow.AddMonths(3).Date
+            };
+            return claimRequest;
+        }
+
+        private static DocumentSubmission BuildDocumentSubmission(
+            DocumentSubmissionWithoutEvidenceRequestRequest request,
+            Claim claim
+        )
+        {
+            var documentSubmission = new DocumentSubmission()
+            {
+                ClaimId = claim.Id.ToString(),
+                State = SubmissionState.Approved,
+                Team = request.Team,
+                ResidentId = request.ResidentId,
+                StaffSelectedDocumentTypeId = request.StaffSelectedDocumentTypeId
+            };
+            return documentSubmission;
+        }
+
+        private static void ValidateRequest(DocumentSubmissionWithoutEvidenceRequestRequest request)
```

Comment:
Is there a reason why we're validating the request this way as opposed to setting up a validator with `FluentValidation`?

Reply:
Addressed. Replaced with FluentValidations.